ui/src/shared/components/FunctionSelector.js

```diff
@@ -0,0 +1,82 @@
+import React, {Component, PropTypes} from 'react'
+import classNames from 'classnames'
```

Comment:
prefer `classnames` capitalization to differentiate from React's native `className` api